### ISSUE TITLE
write tabs' names in 2 rows [1h]

### ISSUE THREAD

**Issue by HelgaMond** [tango-controls/tango-webapp#84]
Make tab's name written in 2 rows. That will essentially simplify the reading of the name and free the space for some other tabs. Please, see the pic and compare two tabs with the new version and the existing one.

![tabs_names](https://user-images.githubusercontent.com/27626862/36529976-eb05e814-17ca-11e8-810a-fb243aead0b8.png)

**Comment by Ingvord** [comment]
```html
<div style="display: table-row;">
  <span style="display: table-cell;vertical-align: middle;">
<img  src= />
    </span>
<span style="display: table-cell;">
  <span style="display: block;">text1</span>
  <span style="display: block;">text2</span>
</span>
<span style="display: table-cell;">x</span>
</div>

```

**Comment by Ingvord** [comment]
Closed due to #120
